indico/htdocs/js/indico/jquery/declarative.js

```diff
@@ -186,4 +187,11 @@
             confirmed.then(execute);
         });
     }
+
+    function setupMathJax() {
+        var $elems = $('.js-mathjax');
+        if ($elems.length) {
```

Comment:
That way if a page has an element with the `js-mathjax`class but the mathjax library is not included in the page this doesn't rise an exception that could prevent other code from running.